testsuite/client-old-server/src/test/java/org/wildfly/client/old/server/util/Version.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright ${year}, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.wildfly.client.old.server.util;
+
+/**
+ * @author Kabir Khan
+ */
+public class Version {
+
+    static final String AS = "jboss-as-";
+    static final String WILDFLY = "wildfly-";
+    static final String EAP = "jboss-eap-";
+
+    public enum AsVersion {
```

Comment:
well 7.0.x stuff we should stop testing.
On related problem, 7.x stuff should be tested with JDK6 not 7....